src/sentry/models/orgauthtoken.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+from django.utils.encoding import force_text
+
+from sentry.db.models import (
+    ArrayField,
+    BaseManager,
+    FlexibleForeignKey,
+    Model,
+    control_silo_only_model,
+    sane_repr,
+)
+from sentry.models.orgauthtokenproject import OrgAuthTokenProject
+
+
+@control_silo_only_model
+class OrgAuthToken(Model):
+    __include_in_export__ = True
+
+    organization = FlexibleForeignKey("sentry.Organization", null=False)
+    projects = models.ManyToManyField(
+        "sentry.Project", related_name="projects", through=OrgAuthTokenProject
+    )
+    # max JWT token length is 2048, +7 for our custom prefix (sntrys_)
+    token = models.CharField(max_length=2055, unique=True, null=False)
+    label = models.CharField(max_length=255, null=False)
+    scope_list = ArrayField(of=models.TextField)
```

Comment:
Should we statically type the list of scopes? Having such a generic field is a bit suspicious.

Reply:
for reference, this is how the scope list is currently stored in other models. But we may improve on this here :D

Reply:
If possible i would like to bound it list of elements, maybe like:

```python
from django.db import models
from django.core.exceptions import ValidationError

class MyModel(models.Model):
    OPTIONS = [
        ('option1', 'Option 1'),
        ('option2', 'Option 2'),
        ('option3', 'Option 3'),
    ]

    choices_array = models.ArrayField(
        models.TextField(),
        validators=[],
    )

    def validate_choices_array(value):
        for choice in value:
            if choice not in dict(MyModel.OPTIONS).keys():
                raise ValidationError(f"{choice} is not a valid choice.")

    choices_array.validators.append(validate_choices_array)
```